### PR TITLE
Get yelp data

### PR DESCRIPTION
**Pull request by csuksangium**

### CODE REVIEW

**Review by Simenhug — APPROVED** [review]
great job

File: testapp/views.py [hunk 1]
```diff
@@ -1,10 +1,45 @@
...
+    # Fall back to Python 2's urllib2 and urllib
+    from urllib2 import HTTPError
+    from urllib import quote
+    from urllib import urlencode
+
+
 # Create your views here.
 
+CLIENT_ID = '_tg2X_Fpcpuae2eNKaHj_w'
+CLIENT_SECRET = 'TLPjiUfjOessjQVEqQpogquNpwNHUkMrba5QCCCASgb5fUfMhwamVNvRvB9tAy78'
```
Comment (freddiev4) [inline testapp/views.py]: I think we should remove these

File: testapp/views.py [hunk 2]
```diff
@@ -1,10 +1,45 @@
...
+
+API_HOST = 'https://api.yelp.com'
+SEARCH_PATH = '/v3/businesses/search'
+BUSINESS_PATH = '/v3/businesses/'  # Business ID will come after slash.
+TOKEN_PATH = '/oauth2/token'
+GRANT_TYPE = 'client_credentials'
+
+DEFAULT_TERM = 'restaurant'
+DEFAULT_LOCATION = 'San Francisco, CA'
```
Comment (freddiev4) [inline testapp/views.py]: Should the default_location be `Boston, MA`?

File: testapp/views.py [hunk 3]
```diff
@@ -23,12 +58,133 @@ def info(request):
...
+
+    print(u'{0} businesses found, querying business info ' \
+        'for the top result "{1}" ...'.format(
+            len(businesses), business_id))
+    response = get_business(bearer_token, business_id)
+    print(u'Result for business "{0}" found:'.format(business_id))
+    pprint.pprint(response, indent=2)
+    return response
+
+def yelpCall(keyTerm, location):
```
Comment (freddiev4) [inline testapp/views.py]: Should we name this `yelp_call()` to be consistent in our code style?

File: testapp/views.py [hunk 4]
```diff
@@ -23,12 +58,133 @@ def info(request):
...
+  return HttpResponse(restName)
 
 def save_user_request(data):
   user_request = UserRequest(location=data.city, term=data.term)
   user_request.save()
 
 def query_yelp(data):
-  pass
\ No newline at end of file
+  pass
```
Comment (freddiev4) [inline testapp/views.py]: Should we remove this `query_yelp()` function? It's not being used

**Review by freddiev4 — CHANGES_REQUESTED** [review]

File: testapp/yelp_clientid.yaml [hunk 5]
```diff
@@ -0,0 +1,2 @@
+CLIENT_ID = '_tg2X_Fpcpuae2eNKaHj_w'
+CLIENT_SECRET = 'TLPjiUfjOessjQVEqQpogquNpwNHUkMrba5QCCCASgb5fUfMhwamVNvRvB9tAy78'
```
Comment (freddiev4) [inline testapp/yelp_clientid.yaml]: Looks like this `.yaml` file still got included in the git history. Can you remove it and fixup all the commits into one commit?

**Review by freddiev4 — CHANGES_REQUESTED** [review]

**Review by freddiev4 — APPROVED** [review]

**Pull request merged**